js/5etools-main.js

```diff
@@ -1498,6 +1498,51 @@ const betteR205etoolsMain = function () {
 				}
 			}
 
+			// Choose and import personallity traits/ideals/bonds/flaws.
+			traits = null;
+			ptrait = null; // Personallity trait
+			ideal = null;
+			bond = null;
+			flaw = null;
+			// Get the JSON for all the tables
+			if (bg.entries) {
+				for (const ent of bg.entries) {
+					if (ent.name && ent.name === "Suggested Characteristics") {
+						traits = ent;
+					}
+				}
+			}
+
+			// Fill the rows 
+			if (traits !== null && traits.entries && traits.entries.length) {
+				for (let i = 0; i < traits.entries.length; i++) {
```

Comment:
this is also fundamentally flawed as `entries` is a tree, not a flat array, so you need to recursively walk the thing to 100% guarantee you'll find the table
Doing so is more hassle than it's worth in this case, so this is fine, but just as a note for future